src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/fsm/SwitchValidateFsm.java

```diff
@@ -203,6 +214,16 @@ public void emitRequests(SwitchValidateState from, SwitchValidateState to, Switc
 
             requestSwitchOfGroups();
 
+            if (sw.get().getFeatures().contains(LAG)) {
+                // at this moment Kilda validated only LAG logical ports
+                Optional<String> ipAddress = sw.map(Switch::getSocketAddress).map(IpSocketAddress::getAddress);
+                if (ipAddress.isPresent()) {
+                    requestLogicalPorts(ipAddress.get());
+                } else {
+                    log.warn("Unable to get IP address of switch {} to get LAGs", switchId);
```

Comment:
It should be represented in final response (i.e. we must "see" that logical ports need to be validated, but validation have failed to do it).